examples/demo_bgp.pp

```diff
@@ -130,46 +130,72 @@
   #---------------------------------------------------------------------------#
```

Comment:
This looks great to me. Go ahead with the new PR and I'll give it one last once-over today and merge it when done.